Avoid:
    mixer.c:153:46: error: implicit declaration of function 'Mix_InitFluidSynth' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

--- mixer.c.orig	2012-01-15 15:01:05.000000000 -0700
+++ mixer.c	2021-04-03 10:50:42.000000000 -0700
@@ -38,6 +38,7 @@
 #include "dynamic_mod.h"
 #include "dynamic_mp3.h"
 #include "dynamic_ogg.h"
+#include "dynamic_fluidsynth.h"
 
 #define __MIX_INTERNAL_EFFECT__
 #include "effects_internal.h"
